src/explore-education-statistics-admin/src/pages/release/data/ReleaseDataFilePage.tsx

```diff
@@ -52,9 +52,25 @@ const ReleaseDataFilePage = ({
   const getReplacementPlanMessage = () => {
     if (replacementDataFileError) {
       return (
-        <WarningMessage>
-          There was a problem loading the data replacement information.
-        </WarningMessage>
+        <>
+          <WarningMessage>
+            There was a problem loading the data replacement information.
+          </WarningMessage>
+
+          {replacementDataFile && (
+            <Button
```

Comment:
This button behaves inconsistently with the one in `DataFileReplacementPlan` (which triggers a modal). We should give them consistent behaviour.

I would suggest hoisting the modal from `DataFileReplacementPlan` into this component. Then use the `onCancel` callback to trigger it, if that makes sense?